src/App.js

```diff
@@ -6,10 +6,95 @@ import "./App.css";
 const App = () => {
   return (
     <div className="App">
-      <header className="App-header">CYF Hotel</header>
+      {/* task 02 */}
+      <Heading />
+      {/* task 3, */}
+      <TouristInfoCards />
       <Bookings />
+      {/* task 4 */}
+      <Footer address={address} />
     </div>
   );
 };
 
+function Heading() {
+  return (
+    <header className="App-header">
+      <img
+        className=".App-logo "
+        src="https://image.flaticon.com/icons/svg/139/139899.svg"
+        alt="hotel's logo"
+      />
+      CYF Hotel
+    </header>
+  );
+}
```

Comment:
this is a good component well done, but extract it out into it's own file & import it into the `App` component. Same with your `TouristInfoCards` component and `Footer` component. This helps with reusability, for example with your `Heading` component at the moment it can only be accessed within `App`, but if you make it it's own component you can import this anywhere you like in your app and use it (you can imagine this would be especially useful with a large scale app)